amivapi/events/queue.py

```diff
@@ -65,7 +65,7 @@ def update_waiting_list(event_id):
 def add_accepted_before_insert(signups):
     """Add the accepted field before inserting signups."""
     for signup in signups:
-        signup['accepted'] = False
+        signup['accepted'] = g.resource_admin and signup.get('accepted', False)
```

Comment:
I think this line needs a comment above, e.g.

```python
# admins may provide a value for `accepted`.
#if not provided or not admin set it to false`.
```

I'm afraid without a comment, nobody will understand what this line is doing.

Maybe it would be useful to just put this in the docstring.